From: Gordon Ball <gordon@chronitis.net>
Date: Sun, 11 Oct 2020 10:35:59 +0000
Subject: test_validator: test jsonschema only

nbformat 5.0.8 added support for (unpackaged) fastjsonschema, and tests
it unconditionally (unlike in the library where it is optional). This
patches the test to bypass testing this alternative validator library.
---
 nbformat/tests/test_validator.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/nbformat/tests/test_validator.py b/nbformat/tests/test_validator.py
index 52936a3..68d2925 100644
--- a/nbformat/tests/test_validator.py
+++ b/nbformat/tests/test_validator.py
@@ -14,6 +14,7 @@ from ..json_compat import VALIDATORS
 
 import pytest
 
+VALIDATORS = ["jsonschema"]
 
 # Fixtures
 @pytest.fixture(autouse=True)
